articles/logic-apps/logic-apps-move-from-mabs.md

```diff
@@ -247,4 +247,4 @@ which lets you synchronize B2B data across integration accounts in different reg
 
 * [What is Logic Apps?](../logic-apps/logic-apps-overview.md)
 * [Create your first logic app](../logic-apps/quickstart-create-first-logic-app-workflow.md), or quickly get started using a [pre-built template](../logic-apps/logic-apps-create-logic-apps-from-templates.md)  
-* [View all the available connectors](../connectors/apis-list.md) that you can use in logic apps
\ No newline at end of file
+* [View all the available connectors](../connectors/apis-list.md) that you can use in logic apps
```

Comment:
I don't detect any changes here in this line?

Reply:
Added a newline.